### DISCUSSION TITLE
SAFE procedure phase 4

### DISCUSSION THREAD

**Discussion by RiannevanderFeen** [asreview/asreview#1857] [Q&A] [answered]
Hello, it is said to add the 10 highest- and lowest ranked (labbelled as irrelevant) records from the previous phase for this phase. The stopping rule again is set at 50 irrelevant records in a row. However, I only added 20 records (10 highest and 10 lowest)! How does that work?

**Comment by Rensvandeschoot** [comment, marked as answer]
The ones you added (10+10) should be labeled records to be used as prior knowledge. Next to these records, you need to add the excluded records from your previous phase, but remove their labels since you want to label them again.
